threagile.yaml

```diff
@@ -1245,6 +1334,7 @@ technical_assets:
     usage: devops
     size: application
     technology: cli
+    out_of_scope: true
```

Comment:
I think it might make sense to leave this in-scope. Argo CD users use the CLI pretty heavily. I'd consider it as important as the UI.

Reply:
Back in scope.